js-getting-started/utils.js

```diff
@@ -1,5 +1,8 @@
 // waiting for code...
 
-function showMessage(message) {
-    document.getElementById('message').textContent = message;
+//changes HERO offer on site
+function showMessage(itemName, discount, price) {
+    document.getElementById('item').textContent = itemName;
+    document.getElementById('discount').textContent = discount;
+    document.getElementById('price').textContent = price;
```

Comment:
Im pretty sure this could be done differently, but don't know how

Reply:
I know, I'll look it up and see what I can find for the future